### PR TITLE
Revert "Feature#4 db setup"

### PR DESCRIPTION
**Pull request by cfcortes**
Reverts BrightCoders-Proyectos/tribo-web#20

According to @YilKanda some routes need to bee fixed before merging

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,22 +2,6 @@
 
 Rails.application.routes.draw do
   get "legal/index"
-  
-  resources :market_places
-  root 'main#index'
```
Comment (YilKanda) [inline config/routes.rb:7]: This command is repeated on line 22, At the time of doing the pull of the master it caused conflict.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (cfcortes) [reply config/routes.rb:7]: @KarinaDell @danielasotomayor99 @EduJimenez96 Can you help us on this?

File: config/routes.rb [hunk 2]
```diff
@@ -2,22 +2,6 @@
...
-      controller: "clearance/passwords",
-      only: [:create, :edit, :update]
-  end
-
-  get "/sign_in" => "clearance/sessions#new", as: "sign_in"
-  delete "/sign_out" => "clearance/sessions#destroy", as: "sign_out"
-  get "/sign_up" => "clearance/users#new", as: "sign_up"
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root to: 'home#index'
   resources :home
```
Comment (YilKanda) [inline config/routes.rb]: At the moment of pulling the master, it marks an error. There are two root

imagen

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by YilKanda — CHANGES_REQUESTED** [review]
Everything that had worked in the database branch was deleted.

imagen

**Pull request merged**